### PR TITLE
[PANGOO-2594][BpkNavigationTabGroup]Fix BPK NavTabGroup Warning

### PR DESCRIPTION
**Pull request by kerrie625**
There is warning For BPK NavTabGroup.So need to fix.

image

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [x] `README.md` (If you have created a new component)
- [x] Component `README.md`
- [x] Tests
- [x] Accessibility tests
    - The following checks were performed:
        - [ ] Ability to navigate using a [keyboard only](https://webaim.org/techniques/keyboard/)
        - [ ] Zoom functionality ([Deque University explanation](https://dequeuniversity.com/checklists/web/text)):
            - [ ] The page SHOULD be functional AND readable when only the text is magnified to 200% of its initial size
            - [ ] Pages must reflow as zoom increases up to 400% so that content continues to be presented in only one column i.e. Content MUST NOT require scrolling in two directions (both vertically and horizontally)
        - [ ] Ability to navigate using a [screen reader only](https://webaim.org/articles/screenreader_testing/)
- [x] Storybook examples created/updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: examples/bpk-component-navigation-tab-group/examples.tsx [hunk 1]
```diff
@@ -92,7 +92,6 @@ const SimpleCanvasDefault = () => (
       onItemClick={() => {}}
       selectedIndex={0}
       type={NAVIGATION_TAB_GROUP_TYPES.CanvasDefault}
-      ariaLabel="Navigation tabs"
```
Comment (amburrrshand) [inline examples/bpk-component-navigation-tab-group/examples.tsx:95]: Why did you decided to remove the ariaLabel?

Comment (kerrie625) [reply examples/bpk-component-navigation-tab-group/examples.tsx:95]: Not remove,  Just change ariaLabel as optional. And this example is used for test props without ariaLabel

File: packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx [hunk 2]
```diff
@@ -49,11 +52,11 @@ export type Props = {
    */
   onItemClick: (e: MouseEvent<HTMLButtonElement | HTMLAnchorElement>,tab: TabItem, index: number) => void;
   selectedIndex: number;
-  ariaLabel: string;
+  ariaLabel?: string;
```
Comment (olliecurtis) [inline packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx:55]: how come this has become optional?

File: examples/bpk-component-navigation-tab-group/examples.tsx (hunk 1 above)
Comment (kerrie625) [reply examples/bpk-component-navigation-tab-group/examples.tsx:95]: When I check header tabs that is not include ariaLabel,so I think it is need to change to option. Is it reasonable for cosumer decide to whether pass this ariaLabel?

Comment (amburrrshand) [reply examples/bpk-component-navigation-tab-group/examples.tsx:95]: In this case, I think it's best practice for accessibility standards to not have it as optional

Comment (kerrie625) [reply examples/bpk-component-navigation-tab-group/examples.tsx]: @olliecurtis @amburrrshand Thank you for your advice. I have reverted the ariaLabel as required.

**Review by amburrrshand — APPROVED** [review]
LGTM

**Pull request merged**